code/modules/client/preference/preferences.dm

```diff
@@ -2080,6 +2067,14 @@ var/global/list/special_role_times = list( //minimum age (in days) for accounts
 					if(href_list["tab"])
 						current_tab = text2num(href_list["tab"])
 
+				if("ambientocclusion")
```

Comment:
This looks like pure binary, on or off, against a new column is really not needed for this